airbyte-integrations/connectors/source-clazar/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "source-clazar"
+description = "Source implementation for clazar."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/clazar"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+packages = [ { include = "source_clazar" }, {include = "main.py" } ]
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
```

Comment:
Please switch this to "< 3.0" and then run `poetry lock` again. That's a mistake on our side — the recent CDK is 2.0+ already. 

```suggestion
airbyte-cdk = "< 3.0.0"
```